src/inmanta/server/agentmanager.py

```diff
@@ -1185,7 +1184,7 @@ async def _ensure_scheduler(
                         autostart_scheduler,
                         self._agent_procs[env],
                     )
-                    await self._stop_scheduler(refreshed_env, connection=connection)
```

Comment:
it this correct? I.e. why let go of the connection?

Reply:
afaics the connection argument was not used in `inmanta. server. agentmanager. AutostartedAgentManager._stop_scheduler`. Am I missing anything ?

Reply:
ok!